test/OpenTelemetry.Exporter.Geneva.Tests/OpenTelemetry.Exporter.Geneva.Tests.csproj

```diff
@@ -1,12 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
   <Import Project="$(RepoRoot)\src\OpenTelemetry.Exporter.Geneva\Common.GenevaExporter.props" />
 
   <PropertyGroup>
-    <Description>Unit test project for Geneva Exporters for OpenTelemetry</Description>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>$(SupportedNetTargets)</TargetFrameworks>
```

Comment:
nit: If the goal is to standardize targets, going from `$(SupportedNetTargets)` to `net8.0` seems like a step backwards 🤷

Reply:
I propose to hardcode `net8.0` for the moment and reintroduce the property when `net6.0` will be completely removed. We can also introduce a new property temporarily now that will only contain `net8.0` if needed but not sure if it's relevant.

Reply:
I understood we don't want to impact all projects at the time and we would prefer to update them one by one. That's why I did it this way.